pkg/scheduler/apis/config/v1alpha2/conversion_test.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha2
+
+import (
+	"k8s.io/utils/pointer"
+	"testing"
+
+	"k8s.io/apimachinery/pkg/conversion"
+	"k8s.io/kube-scheduler/config/v1alpha2"
+	"k8s.io/kubernetes/pkg/scheduler/apis/config"
+)
+
+func TestV1alpha2ToConfigKubeSchedulerConfigurationConversion(t *testing.T) {
+	in := &v1alpha2.KubeSchedulerConfiguration{}
+	out := &config.KubeSchedulerConfiguration{}
+	s := conversion.Scope(nil)
+	if err := Convert_v1alpha2_KubeSchedulerConfiguration_To_config_KubeSchedulerConfiguration(in, out, s); err != nil {
```

Comment:
can you use an scheme as we are doing in v1alpha1/conversion_test.go for #88002

Reply:
Updated to be similar to that test